imports/startup/server/index.coffee

```diff
@@ -168,6 +168,43 @@ Meteor.methods
       """
     makeRequest(query)
 
+  'getTrendingInfectiousAgents': (date, date2, days) ->
+    query = prefixes + """
+      SELECT ?resolvedTerm
+        (sample(?termLabel) as ?word)
+		    (sample(?termLabel2) as ?word2)
+        (count(DISTINCT ?article) as ?count)
+		    (sample(?c2) as ?count2)
+        ((?count2)/(xsd:float(#{escape(days)}*4)) as ?rate2)
+        ((?count)/(xsd:float(#{escape(days)})) as ?rate)
+  		  ((xsd:float(?rate)/(xsd:float(?rate2))) AS ?result)
+      WHERE {
+        ?phrase anno:category "diseases"
+        ; ^dc:relation ?resolvedTerm
+        ; anno:source_doc ?article
+        .
+        ?article pro:post/pro:date ?dateTime .
+        ?resolvedTerm rdfs:label ?termLabel
+        FILTER (?dateTime > "#{escape(date)}"^^xsd:dateTime)
+
+        {SELECT (count(distinct ?article2) as ?c2) ?resolvedTerm ?termLabel2
+    	     WHERE{
+            ?prev_mention anno:source_doc ?article2
+            ; ^dc:relation ?resolvedTerm
+            .
+            ?article2 pro:post/pro:date ?dateTime2 .
+      		  ?resolvedTerm rdfs:label ?termLabel2
+            FILTER (?dateTime2 > "#{escape(date2)}"^^xsd:dateTime)
+           }
+    		   GROUP BY ?resolvedTerm ?termLabel2
+ 		      }
```

Comment:
There are some tab characters here and in other parts of the query that are messing up the indentation.